cmd/config_labels_test.go

```diff
@@ -0,0 +1,140 @@
+// +build linux
+
+package cmd
+
+import (
+	"context"
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/spf13/cobra"
+	fn "knative.dev/kn-plugin-func"
+	"os"
+	"sync"
+	"testing"
+	"time"
+)
+
+type mockFunctionLoaderSaver struct {
+	f fn.Function
+}
+
+func (m *mockFunctionLoaderSaver) Load(path string) (fn.Function, error) {
+	return m.f, nil
+}
+
+func (m *mockFunctionLoaderSaver) Save(f fn.Function) error {
+	m.f = f
+	return nil
+}
+
+func assertLabelEq(t *testing.T, actual fn.Pairs, want fn.Pairs) {
```

Comment:
Perhas a [`reflect.DeepEqual` ](https://pkg.go.dev/reflect#DeepEqual)Here would make this shorter, unless order is not guaranteed...